general/utilities/restart_ioc_when_pv_in_alarm.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.environ["KIT_ROOT"], "ISIS", "inst_servers", "master")))
+from genie_python import genie as g
+from time import sleep
+from server_common.helpers import register_ioc_start
+from typing import List
+from threading import Thread
+
+IOC_TO_RUN_IN = "BGRSCRPT_02"
+register_ioc_start(IOC_TO_RUN_IN)
+g.set_instrument(None)
```

Comment:
As I said before, can you remove these so that they're done in the actual background script rather than in this helper function?

Reply:
This will also mean that you can get rid of some of the imports